.vale/styles/style_guide/Repetition.yml

```diff
@@ -0,0 +1,11 @@
+---
+# Error: style_guide.Repetition.yml
+
+# catch any instances of repeated words. E.g., "This is is a sentence"
+extends: repetition
+message: "'%s' is repeated."
+level: error
+alpha: true
+tokens:
+  - '[^\s]+'
+  
```

Comment:
```suggestion

```

Reply:
I added a blank line at the end of each rule

Reply:
Hard to tell w/ GitHub's interface, but this suggestion is not removing the blank line, just some accidental whitespace.

Reply:
Ohh, makes sense